### DISCUSSION TITLE
GSSOC no. of Projects

### DISCUSSION THREAD

**Discussion by IamRajRohit** [sanjay-kv/Open-source-Practice#896] [Q&A] [answered]
Can we work on more than one issues of multiple projects of different organizations?

**Comment by MrCodYrohit** [comment, marked as answer]
Yes You can work on more than one issues of multiple projects of different organizations and even its better to do so.

**Comment by lovelymahor** [comment]
Yes, you can work on multiple issues simultaneously, whether they are from the same project or different projects. However, it's important to manage your time and workload effectively to ensure you can handle both issues without compromising the quality of your contributions.
